client/my-sites/plans-grid/components/header-price.tsx

```diff
@@ -205,14 +209,38 @@ const PlanFeatures2023GridHeaderPrice = ( {
 					{ isAnyVisibleGridPlanDiscounted && (
 						<Badge className="plan-features-2023-grid__badge" isHidden={ true }></Badge>
 					) }
-					<PlanPrice
-						currencyCode={ currencyCode }
-						rawPrice={ originalPrice.monthly }
-						displayPerMonthNotation={ false }
-						isLargeCurrency={ isLargeCurrency }
-						isSmallestUnit={ true }
-						priceDisplayWrapperClassName="plans-grid-2023__html-price-display-wrapper"
-					/>
+					{ isLargeCurrency ? (
```

Comment:
@chriskmnds I believe if you add the `isAnyVisibleGridPlanDiscounted` condition here it should resolve any spacing oddness we were seeing while still maintaining the original spirit of this PR, which was to fix alignment issues. It seems like we only want the placeholder to exist if _at least_ one other plan is discounted in the grid

```suggestion
					{ isLargeCurrency && isAnyVisibleGridPlanDiscounted ? (
```

Reply:
that does it @jeyip . Thank you :-)

I'm also renaming `shouldShowDiscountedPrice` to `isGridPlanDiscounted` to be consistent with `isAnyVisibleGridPlanDiscounted`